### ISSUE TITLE
URI validation too strict based on type definition

### ISSUE THREAD

**Issue by pramisters** [claeis/ilivalidator#156]
URI ist defined as  TEXT*1023 in the internal INTERLIS model.
Why is the validator more strict and checks something that is neither defined by the type nor by a constraint? From my point of view that ist not correct.

Problem:
There are several MGDMs with attributes of type URI (that we can't change) where we had stored only part of an URI (e.g. in KbS_LV95_V1_3.Belastete_Standorte.Belasteter_Standort.URL_Standort) because the first part of the URI is the same for all sites/objects and can be outsourced to a parameter. Now we have to correct it what increases redundancy.

**Comment by claeis** [comment]
The validation follows the definition of IETF RFC 2396, which is referenced by the URI datatype definition (section 2.8.1 of the INTERLIS 2.3 reference manual).

**Comment by beistehen** [comment]
You are correct when referring to the internal model (see https://www.interlis.ch/modelle/internes-datenmodell). But the RefHB states the following (excerpt from page 43):
![grafik](https://user-images.githubusercontent.com/26199857/48607716-e82b3980-e981-11e8-8268-a9f79accf706.png)
In RFC 2396 it is defined that a scheme and a colon ":" (like `https:` or `ftp:` ) need to be present in the URI. Some rules of INTERLIS are defined in the model(s), but the text of the RefHB belongs to the standard as well.

So to your problem:
What about ignoring the errors concerning your URIs using a configuration that disables validation of the attribute? Or a function that replaces the outsourced parameter before validating the data?
Cheers, Stefan
